### ISSUE TITLE
Yarn install fails because it couldn't find git binary

### ISSUE THREAD

**Issue by rjkz808** [Borales/actions-yarn#15]
As you know, some npm packages can require another ones by git URL. And if I try to install such using `actions-yarn`, it fails with this error:

```bash
[1/4] Resolving packages...
[2/4] Fetching packages...
error Couldn't find the binary git
```

Even if I install git in my workflow this way:

```yml
- name: Preinstall system dependencies
  run: |
    sudo apt update
    sudo apt install git
```

`yarn install` execution still fails.

**Comment by Borales** [comment]
@rjkz808 it happens when one of the dependencies are being fetched directly from the git repo. I'll try to fix this

**Comment by Borales** [comment]
@rjkz808 should be solved with v2.0.2

**Comment by rjkz808** [comment]
> @rjkz808 should be solved with v2.0.2

Ok, thank you!

**Comment by artall64** [comment]
It might be the issue on the sub-dependencies as well like RxJS
